src/core/HTTP/RestClient.ts

```diff
@@ -63,7 +63,8 @@ export default class RestClient {
       baseURL: this.clientUrl,
       headers: this.clientHeaders,
       httpAgent: new HTTPAgent({ keepAlive: true }),
-      maxContentLength: 10 * 1000000, // 10mb is kong file system
+      maxContentLength: Infinity,
```

Comment:
This won't be allowed by the kong service afaik

Reply:
Hi,

What would you recommend then ?

The memory settings are controlled by Kong with the pool memory size and having a hard coded limit on the client side does not work.

10 mb for a full portal including images and JavaScrip assets is not enough.

Advises?

Reply:
The 10mb limit is per file so we should keep it. For larger javascript bundles they should be split into multiple files.

Reply:
The 10mb limit is per file so we should keep it. For larger javascript bundles they should be split into multiple files.